capstone/src/main/webapp/sheetsScript.js

```diff
@@ -62,29 +64,272 @@ async function gatherSheetsInput() {
     }
   }
 
-  const text = await getTextFromSheet(id);
-  handleInput(text, langElement.value, requestedAttributes, tokenizer);
+  // Show general output
+  let sheetNames = await getSheetNames(id);
+  let totalText = '';
+  let range = '!A1:YY';
+  for (const name of sheetNames) {
+    const sheet = await getSpreadsheet(id, name, range);
+    const text = await getTextFromSheet(sheet);
+    totalText += text + '\n';
+  }
+  handleInput(totalText, langElement.value, requestedAttributes, tokenizer);
+
+  // Create spreadsheet if requested
```

Comment:
Could the rest of the code in this method be made its own method? This one is getting a little long and I'm not sure if it's still related to the name `gatherSheetsInput`

Reply:
Yes, sounds good. This is partly because I wanted the functionality to be working first before trying to separate out the code, but I can do that now.